app/views/mou_signatures/_mou_version_1.html.erb

```diff
@@ -1,4 +1,4 @@
-<p>Last updated: 24 October 2023</p>
+<p>Last updated: 06 November 2023</p>
```

Comment:
Is it intentional (or if not, does it matter) that we've got 2 different date formats in this file - `06 November` here and `6 November` on line 909?

Reply:
That's a good question - I'll stick it on the trello card

Reply:
Not a blocker if you want to merge this and resolve that later if its not right

Reply:
Got confirmation for the 06 format so I've updated the PR.

Reply:
Nice - have approved!